zerver/lib/home.py

```diff
@@ -202,6 +213,7 @@ def build_page_params_for_home_page_load(
         show_billing=billing_info.show_billing,
         promote_sponsoring_zulip=promote_sponsoring_zulip_in_realm(realm),
         show_plans=billing_info.show_plans,
+        sponsorship_pending=billing_info.sponsorship_pending,
```

Comment:
Hmm. This field should probably be called `realm_sponsorship_pending` ... but I think we need a more systematic naming pass on these details anyway, so not worth spending time on right now.